nipoppy/env.py

```diff
@@ -9,6 +9,7 @@
 # BIDS
 BIDS_SUBJECT_PREFIX = "sub-"
 BIDS_SESSION_PREFIX = "ses-"
+FAKE_SESSION = "ses-IsFake"
```

Comment:
@nikhil153 @alyssadai @surchs I think we decided to call made-up sessions `ses-unnamed` -- do we still want to go with that name or use something else?